ACE/include/makeinclude/platform_g++_common.GNU

```diff
@@ -74,6 +74,20 @@ endif
 
 CXX_FULL_VERSION := $(shell $(CXX_FOR_VERSION_TEST) --version)
 
+# Minimum C++ level is now C++17, gcc until version 11 have an older version as default
```

Comment:
This isn't consistent with what the title of PR is. I'm also still not sure what the purpose of changing the default from the compiler default is. Does ACE still compile with C++14?

Reply:
ACE master currently compiles with C++14, this is a first step towards C++17 as minimum required level. Someone has to dive into the microsoft support, when we have there C++17 as default I plan to move C++17 as minimum level